examples/platform_channel_swift/ios/Runner/AppDelegate.swift

```diff
@@ -13,19 +27,21 @@ import Flutter
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
     GeneratedPluginRegistrant.register(with: self)
-    let controller: FlutterViewController = window?.rootViewController as! FlutterViewController
-    let batteryChannel = FlutterMethodChannel(name: "samples.flutter.io/battery",
+    guard let controller = window?.rootViewController as? FlutterViewController else {
+      return false
```

Comment:
Returning false here is something you'd do if you're doing deep linking and don't wanna handle the URL that's passed. In this case, if the view controller isn't what you want, you should `fatalError("rootViewController is not type FlutterViewController")`

[Examples of using fatalError in IOSched-iOS](https://github.com/google/iosched-ios/search?q=fatal&unscoped_q=fatal) if you need to be reassured.

Reply:
Done.